nodejs/eks/cluster.ts

```diff
@@ -126,9 +126,9 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
             args.tags,
             args.clusterSecurityGroupTags,
         ]).apply(([tags, clusterSecurityGroupTags]) => (<aws.Tags>{
-            "Name": `${name}-eksClusterSecurityGroup`,
-            ...tags,
             ...clusterSecurityGroupTags,
+            ...tags,
+            "Name": `${name}-eksClusterSecurityGroup`,
```

Comment:
Didn’t we in other places agree that allowing defaults to be overridden by user-provided values makes sense for tags?

Reply:
Yes, this is just an artifact of an old commit that I hadn't updated upstream. We allow `tags` to override defaults on all taggable resources.